### ISSUE TITLE
core dumped!

### ISSUE THREAD

**Issue by donfisico** [jrnewell/spotify-ripper#215]
Hi, I use ubuntu 14.04 x64 OS, I followed the installations instructions without problems. When I tried to run spotify-ripper, as specifed:
$spotify-ripper -u _myuser _-p _mypassword_ spotify:track:2zYzyRzz6pRmhPzyfMEC8s
I got the classic annoying message of "Segmentation fault  core dumped" .
I can upload the core file to anyone that is willing to help me.

**Comment by hover20** [comment]
Currently I'm also stuck at this point. Workaround:

Use "sudo spotify-ripper...."

**Comment by seandex** [comment]
sudo: spotify-ripper: command not found 
:(
what am I missing? other commands are working fine.

**Comment by hover20** [comment]
Are you typing "sudo spotify-ripper" or "sudo: spotify-ripper"? First command is correct.

For example "sudo spotify-ripper -u user spotify:track:52xaypL0Kjzk0ngwv3oBPR"

But always remember: Executing a command with sudo means "Execute as superuser e.g. root". Please only use this extension if you are sure what you are doing :)

**Comment by donfisico** [comment]
Thanks hover20, it **almost** works with "sudo spotify-ripper" , almost.... the problem NOW is that :

> Logging in...
> Logged in failed: USER_NEEDS_PREMIUM
> Encountered issue while logging into Spotify, aborting...
wich is surprising, I never read before that you need to be a premium user to use spotify ripper!!!!

**Comment by Fiddlesteak** [comment]
well... you already need a spotify premium account for the spotify_appkey.key so

**Comment by hover20** [comment]
See Wiki of Spotify-Ripper

You are able to create your personal developer key here:  https://devaccount.spotify.com/login/

**Comment by Fiddlesteak** [comment]
yes but you need to log in with a premium acc on https://devaccount.spotify.com/login/ too I thought

**Comment by donfisico** [comment]
Fiddlesteak: you are everywhere, don't you?
